server/project.go

```diff
@@ -635,6 +635,18 @@ func (s *Server) registerProjectRoutes(g *echo.Group) {
 	})
 }
 
+func validateProjectKey(key string) *echo.HTTPError {
+	if key == "" {
+		return echo.NewHTTPError(http.StatusBadRequest, "Project key can not be empty")
+	}
+	for _, l := range key {
```

Comment:
unit test, or use regex in this case since it's simple enough.

Reply:
Done.